docs/environment.md

```diff
@@ -0,0 +1,53 @@
+# Environment development
+
+This is a practical guide for developers wanting to develop new environments, or integrate existing third-party environments, within cogment verse.
+
+At the moment, all environment implementations are provided by a single service
+with multiple endpoints. This is likely to change in the future and the documentation
+below will need to be updated to reflect any changes in the architecture.
+
+## Method 1: Add a new endpoint to the existing environment service
+
+### Step 1 - Update dependencies
+
+If your environment has any additional dependencies, you will need to update the [`Dockerfile`](/environment/Dockerfile) and [`pyproject`](/environemtn/pyproject.toml)
```

Comment:
```suggestion
If your environment has any additional dependencies, you will need to update the [`Dockerfile`](/environment/Dockerfile) and [`pyproject`](/environment/pyproject.toml)
```

I'm not sure you need to do anything to the Dockerfile in this case.

Also you can add a link to https://github.com/cogment/cogment-verse/blob/main/docs/development_setup.md#dependencies that provides more in depth instructions on the dependencies management.

Reply:
Added the link and some clarification. It may still be necessary in some cases (or at least more convenient) to modify the Dockerfile for system dependencies (e.g. Mujoco).